pyrato/parametric.py

```diff
@@ -171,6 +104,13 @@ def air_attenuation_coefficient(
         The resulting attenuation coefficient.
 
     """
+    from pyfar.classes.warnings import PyfarDeprecationWarning
+    import warnings
+
+    warnings.warn(
+        'Will be replaced by respective function in pyfar before v1.0.0',
+        PyfarDeprecationWarning)
+
```

Comment:
```suggestion
```

this then belongs in a pr merging into ``main``/``develop`` and not into ``develop_v1.0.0``

Reply:
No, this is just so we don't forget to change it. ;)